<!doctype html>
<html lang="en">
  
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script>
    $(document).ready(function(){
      // Add smooth scrolling to all links
      $("a").on('click', function(event) {
    
        // Make sure this.hash has a value before overriding default behavior
        if (this.hash !== "") {
          // Prevent default anchor click behavior
          event.preventDefault();
    
          // Store hash
          var hash = this.hash;
    
          // Using jQuery's animate() method to add smooth page scroll
          // The optional number (800) specifies the number of milliseconds it takes to scroll to the specified area
          $('html, body').animate({
            scrollTop: $(hash).offset().top
          }, 200, function(){
       
            // Add hash (#) to URL when done scrolling (default click behavior)
            window.location.hash = hash;
          });
        } // End if
      });
    });
    </script>
    <head>
    <style>
      
    .css-selector
{
    font-family: 'Merriweather', serif;
}
  
    .footer {
  
   left: 0;
   bottom: 0;
   width: 100%;
   background-color: grey;
   color: white;
   text-align: center;
}
#customers {

font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
border-collapse: collapse;
width: 70%;
}

#customers td, #customers th {
border: 1px solid #ddd;
padding: 8px;
}

#customers tr:nth-child(even){background-color: #f2f2f2;}
#customers tr:nth-child(odd){background-color: #adffc3;}
#customers tr:hover {background-color: #ddd;}

#customers th {
padding-top: 12px;
padding-bottom: 12px;
text-align: left;
background-color: #4CAF50;
color: white;
}
		
.footer {
  
   left: 0;
   bottom: 0;
   width: 100%;
   background-color: grey;
   color: white;
   text-align: center;
}
      </style>
    <title>About &mdash; CASTLE </title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link href="https://fonts.googleapis.com/css?family=Muli:400,700|Hepta+Slab:400,700&display=swap" rel="stylesheet">

    <link rel="stylesheet" href="fonts/icomoon/style.css">

    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/bootstrap-datepicker.css">
    <link rel="stylesheet" href="css/jquery.fancybox.min.css">
    <link rel="stylesheet" href="css/owl.carousel.min.css">
    <link rel="stylesheet" href="css/owl.theme.default.min.css">
    <link rel="stylesheet" href="fonts/flaticon/font/flaticon.css">
    <link rel="stylesheet" href="css/aos.css">

    <link href="https://fonts.googleapis.com/css?family=Merriweather&display=swap" rel="stylesheet"> 
    <!-- MAIN CSS -->
    <link rel="stylesheet" href="css/style.css">

  </head>

  <body data-spy="scroll" data-target=".site-navbar-target" data-offset="300">

    
    <div class="site-wrap" id="home-section">

      <div class="site-mobile-menu site-navbar-target">
        <div class="site-mobile-menu-header">
          <div class="site-mobile-menu-close mt-3">
            <span class="icon-close2 js-menu-toggle"></span>
          </div>
        </div>
        <div class="site-mobile-menu-body"></div>
      </div>



      <header class="site-navbar site-navbar-target" role="banner">

        <div class="container">
          <div class="row align-items-center position-relative">

            <div class="col-3 ">
              <div class="site-logo">
                <a href="index.html" class="font-weight-bold"><img src="images/newlogo.png" height="80px" width="90px"></a>
              </div>
            </div>

            <div class="col-9  text-right">
              

              <span class="d-inline-block d-lg-none"><a href="#" class="text-white site-menu-toggle js-menu-toggle py-5 text-white"><span class="icon-menu h3 text-white"></span></a></span>

              

              <nav class="site-navigation text-right ml-auto d-none d-lg-block" role="navigation">
                <ul class="site-menu main-menu js-clone-nav ml-auto ">
                  <li><a href="index.html" class="nav-link">Home</a></li>
                  <li class="active"><a href="about.html" class="nav-link">About</a></li>
                 
                  <li><a href="contact.html" class="nav-link">Contact</a></li>
                </ul>
              </nav>
            </div>

            
          </div>
        </div>

      </header>

    <div style="font-family: 'Merriweather',serif;"  class="ftco-blocks-cover-1">
      <div class="site-section-cover overlay" style="background-image: url('images/about.png')">
        <div class="container">
          <div class="row align-items-center">
            <div class="col-md-7">
              <h1 class="mb-3 text-primary"><font color="black">About CASTLE</h1>
              <p>CASTLE is a collection of synthetic lethal data for various pathogenic organisms, generated computationally. CASTLE is freely available for non-commercial use. It is maintained by the <a href="https://home.iitm.ac.in/kraman/lab/"  target="_blank"><font color="darkblue">Computational Systems Biology Lab</font></a> at the Indian Institute of Technology, Madras.</font> </p>
              <p><a href="#Synthetic_Lethals" class="btn btn-primary">Read More</a></p>
            </div>
          </div>
        </div>
      </div>
    </div>


  
    

   <div id="Synthetic_Lethals" style="font-family: 'Merriweather',serif;"  class="site-section bg-black about-me">
     <div class="container">
       <div class="row align-items-center">
         <div class="col-md-6 mb-5 mb-md-0">
           <img src="images/sl1.jpg" alt="Image" class="img-fluid">
         </div>
         <div class="col-md-5 ml-auto">
           <h3 class="text-white mb-5">Synthetic lethals</h3>
           <blockquote class="quote-29281">
            <p style="text-align:justify"><font color="white">Synthetic Lethals are sets of reactions/genes, in which only the simultaneous deletion of all reactions/genes in the set will abate growth and lead to the subsequent death of the organism <sup>[1]</sup>. Figure 1 below, outlines the logic of synthetic lethality: 
              When there are two genes/reactions in a given set, the deletion of both reactions/genes is required to kill the organism.
            </font> </p>
           </blockquote>
           <p style="text-align:justify"><font color="white">The lethals in this database were identified using an <i>in silico</i> approach, applying a parallel version of the Fast-SL algorithm <sup>[1]</sup> (for MATLAB) to genome-scale metabolic network reconstructions taken from the <a href="http://bigg.ucsd.edu/" target="_blank"><u>BIGG</u></a> and <a href="https://www.vmh.life/" target="_blank"><u>VMH</u></a> databases. The Fast-SL algorithm is available at: <a href="https://github.com/RamanLab/FastSL" target="_blank"><u>https://github.com/RamanLab/FastSL</u></a><sup> [1]</sup>.</p>
           </font>
           <p style="text-align:justify"><font color="white">
            The CASTLE database provides single, double, and triple reaction(s)/gene(s) lethal data for several different pathogenic organisms, available in three different formats- JSON, CSV, and MAT. Along with this data, information about the organism’s growth rate and links to the original databases from which the reconstructions were taken are also provided. </p>
           
           </div></font>

         
         </div>
       </div>
     </div>
   </div>
  

    <div class="site-section">
      <div class="container">
        <div class="row mb-5 ">
          <div style="font-family: 'Merriweather',serif;" class="col-md-7 text-center mx-auto">
            <h2 class="serif">How do I use CASTLE?</h2>
          </div>
        </div>
        <div style="font-family: 'Merriweather',serif;" class="row justify-content-center">
          <div style="font-family: 'Merriweather',serif;" class="col-md-8">

           
            <p style="text-align:justify">
                The full list of available organisms can be browsed from the home page and the search bar can be used to find specific organisms. Beside the organism’s name, the number of each type of lethal is displayed. Each of these numbers correspond to a table containing the specific reactions/genes. Each of these reactions/genes can be selected to reveal supplementary information about that reaction/gene. The following convention is used:
                
              </p>
              <br><br>
<table id="customers" align="center">
  <tr>
    <th><center>Convention</center></th>
    <th><center>Meaning</center></th>
   
  </tr>
  <tr>
    <td><center>Jsl</center></td>
    <td><center>Single Reaction Lethals</center></td>
  </tr>
  <tr>
    <td><center>Jdl</center></td>
    <td><center>Double Reaction Lethals</center></td>
  </tr>
  <tr>
    <td><center>Jtl</center></td>
    <td><center>Triple Reaction Lethals</center></td>
  </tr>
  <tr>
    <td><center>Sgd</center></td>
    <td><center>Single Gene Lethals</center></td>
  </tr>
  <tr>
    <td><center>Dgd</center></td>
    <td><center>Double Gene Lethals</center></td>
  </tr>
  <tr>
    <td><center>Tgd</center></td>
    <td><center>Triple Gene Lethals</center></td>
    
  </tr>
</table>
              <br><br>
            
              <p style="text-align:justify">Next to the organism’s name, is an icon, which can be selected to download a ZIP file containing the lethal data in all the available formats. <br><br>On each organism’s page, the links in the table can be selected to display the specific genes/reactions identified for each type of lethal. Also, in the table, are icons which represent the available formats for each type of lethal.  These icons can be selected to download the respective lethal information. </p>
              <br>
              <h3 class="serif"><font color="#fca400">Bibliography</font></h2>
             
               
                <p style="text-align:justify">[1] Pratapa, A., Balachandran, S. & Raman, K. Fast-SL: an efficient algorithm to identify synthetic lethal sets in metabolic networks. Bioinformatics 31, 3299–3305 (2015).
                  <br>
                  [2] Nijman, S. M. Synthetic lethality: General principles, utility and detection using genetic screens in human cells. FEBS Letters 585, 1–6 (2010).
                  <br>
                  [3] Raman, K., Pratapa, A., Mohite, O. & Balachandran, S. Computational Prediction of Synthetic Lethals in Genome-Scale Metabolic Models Using Fast-SL. Methods in Molecular Biology Metabolic Network Reconstruction and Modeling 315–336 (2017). doi:10.1007/978-1-4939-7528-0_14</p>
                
                  <h3 class="serif"><font color="#fca400">Credits</font></h2>
             
                    <p style="text-align:justify">
                      This database was developed by the combined efforts of Arjun Raghavan ( Student of McMaster University, Hamilton, Canada [Email Id: raghavaa@mcmaster.ca] ) and S.Sunanda ( M.Tech Bioinformatics, SASTRA Deemed To Be University, Thanjavur ) under the guidance of Dr.Karthik Raman.<br>
                       <br>
                       Images Courtesy : Wikipedia
                      </div>
                      
        </div>
      </div>
    </div>
    <div class="footer">
      <p><p align="center"> Copyright &copy; CASTLE <script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart text-danger" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a></p></p>
    </div>
  </div>
  

    

    <script src="js/jquery-3.3.1.min.js"></script>
    <script src="js/jquery-migrate-3.0.0.js"></script>
    <script src="js/popper.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/owl.carousel.min.js"></script>
    <script src="js/jquery.sticky.js"></script>
    <script src="js/jquery.waypoints.min.js"></script>
    <script src="js/jquery.animateNumber.min.js"></script>
    <script src="js/jquery.fancybox.min.js"></script>
    <script src="js/jquery.stellar.min.js"></script>
    <script src="js/jquery.easing.1.3.js"></script>
    <script src="js/bootstrap-datepicker.min.js"></script>
    <script src="js/isotope.pkgd.min.js"></script>
    <script src="js/aos.js"></script>

    <script src="js/main.js"></script>

  </body>

</html>

